task/common/machine/script.go

```diff
@@ -32,6 +32,15 @@ base64 --decode << END | sudo tee /usr/bin/tpi-task > /dev/null
 END
 chmod u=rwx,g=rx,a=rx /usr/bin/tpi-task
 
+sudo tee /usr/bin/tpi-task-shutdown << 'END'
+#!/bin/bash
+if ! test -z "$CI"; then
+  cml rerun-workflow
```

Comment:
yikes! Super counter-intuitive "feature" which the user cannot disable?

Reply:
Yes, its the only way right now effective with:
 - timed out workflow
 - In other CIs a simple loop  would work but its expensive having the CI runner also running

Reply:
Speaking about the capability of managing the model afterwards within the CI and second be able to destroy the task.
Things that could be somehow mitigated:

 - Artefacts can be managed within  the task as in the CI simply because we pass the CI env.
 - Design a kind of a mechanism to destroy the tasks